src/App.scss

```diff
@@ -20,13 +20,13 @@
   background-size: cover;
   margin: 0;
   padding: 0;
-  margin-top: -45px;
   min-height: 100vh;
 }
 
 .LoginComponent, .ComponentWrapper {
   background: #fff;
   padding: 50px;
+  margin: 50px;
```

Comment:
After update, inner window was bumping the top "nav bar".  Add margin so we can still see some of the blue background above the white panel.